modules/web/src/app/settings/admin/defaults/static-labels-form/component.ts

```diff
@@ -51,11 +52,13 @@ export class StaticLabelsFormComponent implements OnInit, OnDestroy {
   ngOnInit(): void {
     this._initForm();
 
-    this.staticLabels?.forEach(label => {
-      this._addStaticLabel(label);
-    });
-    this._addStaticLabel();
-    this._updateLabelsObject();
+    setTimeout(() => {
```

Comment:
Why do we need `setTimeout`? Can we handle this in `ngOnChanges` if we want to wait for the value of `staticLabels`?

Reply:
well i wasn't sure what to use here `ngOnChanges` or just `setTimeout`, but i thought `setTimeout` is simpler and i only need the static labels just for one time.
you think `ngOnChanges` is better ?

Reply:
`ngOnChanges` is better because we can be sure that `staticLabels` is not empty. With `setTimeout` we're just guessing that `staticLabels` will have a value in 500ms.